### ISSUE TITLE
Error: estimator MLPClassifier is not supported

### ISSUE THREAD

**Issue by frankicia** [TeamHG-Memex/eli5#395]
Hello,

I've trying to use eli5 in a MLPClassifier.
I encounter the following error:

` Error: estimator MLPClassifier(max_iter=3000, random_state=1010) is not supported `

The code that gives the error is:
```
mlp = joblib.load(dirModels+fileName)
eli5.show_weights(mlp, top=10)
```

I've gone through the tutorial in https://eli5.readthedocs.io/en/latest/tutorials/sklearn-text.html with no issues.

**Comment by lopuhin** [comment]
Thanks for the report, it is indeed not supported and it's a non-trivial task to support it with ``show_weights``, might need an API change as well to support passing a sample of documents.

**Comment by zeromh** [comment]
@lopuhin Is ELI5 still being updated and supported? It looks like it isn't. Asking because I need to decide whether to use ELI5 in my next project or move on to something else.

**Comment by lopuhin** [comment]
@zeromh maintenance and PR review should resume next week, sorry for lack of support recently.

**Comment by lopuhin** [comment]
@zeromh But in the context of this request - I personally don't have MLP show_weights support in mind in near future, so if you found a library which has it, I'd suggest using it (and would be glad if you share a link).
